{
  "id": "uno-ooo-parser",
  "version": "0.1.20",
  "libre_office_ver": "7.2",
  "name": "GridInvalidModelException",
  "type": "exception",
  "namespace": "com.sun.star.awt.grid",
  "parser_args": {
    "sort": true,
    "long_names": true,
    "remove_parent_inherited": true
  },
  "writer_args": {
    "include_desc": true
  },
  "data": {
    "allow_db": true,
    "from_imports": [
      [
        "...uno.runtime_exception",
        "RuntimeException",
        "RuntimeException_d7390ced"
      ]
    ],
    "from_imports_typing": [],
    "extends_map": {
      "com.sun.star.uno.RuntimeException": "RuntimeException_d7390ced"
    },
    "quote": [],
    "typings": [],
    "requires_typing": false,
    "full_imports": {
      "general": [
        "com.sun.star.uno.RuntimeException"
      ],
      "typing": []
    },
    "name": "GridInvalidModelException",
    "imports": [],
    "namespace": "com.sun.star.awt.grid",
    "extends": [
      "com.sun.star.uno.RuntimeException"
    ],
    "desc": [
      "Exception is thrown when data or column model isn't set.",
      "",
      "**since**",
      "",
      "    OOo 3.3"
    ],
    "url": "https://api.libreoffice.org/docs/idl/ref/exceptioncom_1_1sun_1_1star_1_1awt_1_1grid_1_1GridInvalidModelException.html",
    "items": {}
  }
}